content/dstu2/patient.md

```diff
@@ -7,22 +7,30 @@ title: Patient | DSTU 2 API
 * TOC
 {:toc}
 
+## Terminology Bindings
+
+<%= terminology_table(:patient, :dstu2) %>
+
 ## Search
 
 Search for Patients that meet supplied query parameters:
 
     GET /Patient?:parameters
 
+_Implementation Notes_
+
+* The [animal] and [link] Is-Modifier fields are not supported and will not be returned.
+
 ### Parameters
 
- Name                                                    | Required? | Type                                                     | Description
----------------------------------------------------------|----------------------------------------------------------|-----------------------------------------------------------------------------------
-`_id`                                                    | N, if populated all other parameters are ignored | [`token`](http://hl7.org/fhir/DSTU2/search.html#token)  | The logical resource id associated with the resource.
-`birthdate`                                              | At least one parameter must be populated other than `_count` | [`date`](http://hl7.org/fhir/DSTU2/search.html#date)    | The patient's date of birth.  Example: `1961-01-16`
-`identifier`                                             | At least one parameter must be populated other than `_count` | [`token`](http://hl7.org/fhir/DSTU2/search.html#token)  | A patient identifier.  Example: `01022228`
-`name`                                                   | At least one parameter must be populated other than `_count` | [`string`](http://hl7.org/fhir/DSTU2/search.html#string)| A portion of either family or given name of the patient. Example: `Peters`
-`telecom`                                                | At least one parameter must be populated other than `_count` | [`token`](http://hl7.org/fhir/DSTU2/search.html#token)  | The value in any kind of telecom details of the patient. Example: `(816) 475-2374`
-[`_count`](http://hl7.org/fhir/DSTU2/search.html#count)| N | [`number`](http://hl7.org/fhir/DSTU2/search.html#number)| The maximum number of results to return. Defaults to `20`.
+ Name        | Required?                                                    | Type       | Description
+-------------|--------------------------------------------------------------|------------|------------------------------------------------------------
+`_id`        | No, if populated all other parameters are ignored            | [`token`]  | The logical resource id associated with the resource.
```

Comment:
Can we change the comma after No to a period? As is it trips me up because I keep expecting everything after the comma to be additional qualifications/information about the No when in actuality it is an unrelated thought.